### PR TITLE
WIP: Trying with the Debian way for PECL packages

### PR DESCRIPTION
**Pull request by szepeviktor**
preparations for https://github.com/nbs-system/snuffleupagus/issues/274

See https://manpages.debian.org/stretch/dh-php/dh_php.1.en.html#FILES

### CODE REVIEW

**Comment by szepeviktor** [comment]
I think we need to make the repo like a real PHP extension.
e.g. https://github.com/phpredis/phpredis/

**Comment by szepeviktor** [comment]
And name our package `php-snuffleupagus`

File: debian/rules [hunk 1]
```diff
@@ -1,33 +1,14 @@
...
-%:
-	dh $@ 
-
-override_dh_auto_clean:
-	cd ./src; phpize --clean
-
-override_dh_auto_configure:
-	cd ./src; phpize
-	cd ./src; ./configure --enable-snuffleupagus
+PECL_CONFIGURE_MAINT_APPEND = --enable-snuffleupagus
```
Comment (jvoisin) [inline debian/rules]: We can use `PECL_NAME_OVERRIDE` to rename the package to php-snuffleupagus if you want :)

Comment (szepeviktor) [reply debian/rules]: Okay

**Comment by szepeviktor** [comment]
@jvoisin How about structuring the repo as a PECL package?

e.g. https://github.com/phpredis/phpredis/

**Comment by jvoisin** [comment]
As in dumping every `.c` and `.h` into the root directory? I'm not fan of having ~50 files lying there :/

**Comment by szepeviktor** [comment]
I think all PECL packages look like that.

I hope you will like it!

**Comment by jvoisin** [comment]
@kkadosh @buixor can I have your opinion on this one?

**Comment by jvoisin** [comment]
Friendly ping.

**Comment by jvoisin** [comment]
Friendly ping.

**Comment by jvoisin** [comment]
Friendly ping, again.

**Comment by buixor** [comment]
I'm not found neither of having all the files in the main directory neither.
Asking around if there is an alternative :)

**Comment by wargio** [comment]
I don't think it's required to have everything in the root folder.

**Pull request closed without merging**